src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/GovNotifyEmailService.kt

```diff
@@ -55,3 +57,49 @@ class CourtNewBookingEmail(
     addPersonalisation("postAppointmentInfo", postAppointmentInfo ?: "Not required")
   }
 }
+
+class NewCourtBookingPrisonCourtEmail(
+  address: String,
+  prisonerFirstName: String,
+  prisonerLastName: String,
+  prisonerNumber: String,
+  date: LocalDate = LocalDate.now(),
+  court: String,
+  courtEmailAddress: String,
+  prison: String,
+  preAppointmentInfo: String?,
+  mainAppointmentInfo: String,
+  postAppointmentInfo: String?,
+  comments: String?,
+) : Email(address, prisonerFirstName, prisonerLastName, prisonerNumber, date, comments) {
+  init {
+    addPersonalisation("court", court)
+    addPersonalisation("courtEmailAddress", courtEmailAddress)
+    addPersonalisation("prison", prison)
+    addPersonalisation("preAppointmentInfo", preAppointmentInfo ?: "Not required")
+    addPersonalisation("mainAppointmentInfo", mainAppointmentInfo)
+    addPersonalisation("postAppointmentInfo", postAppointmentInfo ?: "Not required")
+  }
+}
+
+class NewCourtBookingPrisonNoCourtEmail(
+  address: String,
+  prisonerFirstName: String,
+  prisonerLastName: String,
+  prisonerNumber: String,
+  date: LocalDate = LocalDate.now(),
+  court: String,
+  prison: String,
+  preAppointmentInfo: String?,
+  mainAppointmentInfo: String,
+  postAppointmentInfo: String?,
+  comments: String?,
+) : Email(address, prisonerFirstName, prisonerLastName, prisonerNumber, date, comments) {
+  init {
+    addPersonalisation("court", court)
+    addPersonalisation("prison", prison)
+    addPersonalisation("preAppointmentInfo", preAppointmentInfo ?: "Not required")
+    addPersonalisation("mainAppointmentInfo", mainAppointmentInfo)
+    addPersonalisation("postAppointmentInfo", postAppointmentInfo ?: "Not required")
```

Comment:
I did wonder whether we need to replicate all these 22 templates, or whether we could make them "smarter" - with replaceable sections based on which data we had? Depends on the differences between the templates though.

Reply:
Maybe copy for now, and exploit any commonalities when we see them?

Reply:
Yeah 22 is a lot of templates.  Keen to refine/slim down.